indico/modules/events/papers/controllers/management.py

```diff
@@ -51,3 +54,32 @@ def _process(self):
             logger.info("Paper managers of %r updated by %r", self.event_new, session.user)
             return jsonify_data(flash=False)
         return jsonify_form(form)
+
+
+class RHSwitchReviewingType(RHManagePapersBase):
+
+    CSRF_ENABLED = True
+
+    def _process_PUT(self):
+        event = self.event_new
```

Comment:
I'd just use `self.event_new` - eventually `event_new` will be renamed to `event` and having `event = self.event` is a bit unnecessary ;)